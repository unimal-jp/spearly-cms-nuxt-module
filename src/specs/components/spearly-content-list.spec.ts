import { shallowMount, Wrapper } from '@vue/test-utils'
import SpearlyContentList from '../../components/spearly-content-list.vue'

describe('SpearlyContentList', () => {
  let wrapper: Wrapper<SpearlyContentList>

  beforeEach(() => {
    wrapper = shallowMount(SpearlyContentList, {
      propsData: { id: 'CONTENT_TYPE_ID' },
    })
  })

  describe('initialized', () => {
    it('snapshot', () => {
      expect(wrapper.element).toMatchSnapshot()
    })
  })
})
