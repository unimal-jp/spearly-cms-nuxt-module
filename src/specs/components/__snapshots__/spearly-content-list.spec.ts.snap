// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SpearlyContentList initialized snapshot 1`] = `<div />`;
